src/V3Param.cpp

```diff
@@ -114,6 +244,8 @@ class ParamVisitor : public AstNVisitor {
 
     string m_generateHierName;  // Generate portion of hierarchical name
 
+    ParameterizedHierBlocks m_hierBlocks;
```

Comment:
Need comment.  Avoid extra newlines between member variables unless really needed for readability.

Reply:
Done in e15f7832f46458bb0c7d949453707e0e64d21ed4.

`m_generateHierName` is not related to hierarchical Verilation, So
- newline between `m_generateHierName` and `m_hierBlocks` is left.
- The explanation of `m_generateHierName` uses `hierarchy` instead of `hierarchical`.